### PR TITLE
Update Julia wrappers for CUDA v12.5

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

File: res/wrap/Manifest.toml [hunk 1]
```diff
@@ -352,10 +352,10 @@ uuid = "83775a58-1f1d-513f-b197-d71354ab007a"
 version = "1.2.13+1"
 
 [[cuQuantum_jll]]
-deps = ["Artifacts", "CUDA_Runtime_jll", "CUTENSOR_jll", "JLLWrappers", "LazyArtifacts", "Libdl", "Pkg", "TOML"]
-git-tree-sha1 = "decd8787d8e93a627b9757c4ef82e9e7737becda"
+deps = ["Artifacts", "CUDA_Runtime_jll", "CUTENSOR_jll", "JLLWrappers", "LazyArtifacts", "Libdl", "TOML"]
+git-tree-sha1 = "7051c53a296f13fb1f944bc9873f43c20301d874"
 uuid = "b75408ef-6fdf-5d74-b65a-7df000ad18e6"
-version = "22.11.13+0"
+version = "24.3.0+0"
```
Comment (maleadt) [inline res/wrap/Manifest.toml]: Please keep the update limited, by only doing `]up CUDA_SDK_jll`.

File: lib/cudadrv/libcuda.jl [hunk 2]
```diff
@@ -1145,23 +1163,23 @@ end
 
 const CUasyncNotificationType = CUasyncNotificationType_enum
 
-struct var"##Ctag#299"
```
Comment (maleadt) [inline lib/cudadrv/libcuda.jl:1148]: Did you run this in a fresh session, i.e., `julia --project res/wrap.jl`? If not, that should hopefully remove some of these changes.

**Review by maleadt — CHANGES_REQUESTED** [review]
Should have realized the wrapper bump in my PR didn't had enough changes...

File: lib/cudadrv/libcuda.jl (hunk 2 above)
Comment (amontoison) [reply lib/cudadrv/libcuda.jl:1148]: I did:
```shell
cd CUDA.jl/res/wrap
julia --project
]up
include("wrap.jl")
main()
```

I just updated the PR to only update `CUDA_SDK_jll`.

**Comment by amontoison** [comment]
The tests are failing because of `NVML_STRUCT_VERSION`.
I will update `nvml.toml` to add additional constants [here](https://github.com/JuliaGPU/CUDA.jl/blob/master/res/wrap/nvml.toml#L7).

**Comment by amontoison** [comment]
Tests only passed on Julia 1.9.
It's already the case on `master`.
I don't know if we have additional errors with this PR.

**Review by maleadt — APPROVED** [review]

**Pull request merged**